README.md

```diff
@@ -1,2 +1,2 @@
 # HW01
-Edit the readme
+This file has been edited.
```

Comment:
Nice! Your edit means that you were able to pull the original file and then commit and push your changes, which was a goal of the assignment.